docs/Inputs.md

```diff
@@ -662,17 +667,6 @@ import { AutocompleteInput, ReferenceInput } from 'react-admin';
 </ReferenceInput>
 ```
 
-Lastly, would you need to override the props of the suggestion's container (a `Popper` element), you can specify them using the `options.suggestionsContainerProps`. For example:
```

Comment:
Since `suggestionsContainerProps` was introduced by react-admin, shouldn't we clarify that now, `@mui` way of overriding the `Popper` container should be used?

Reply:
That's in the MUI documentation, I don't think we should include it